fhttp/http_test.go

```diff
@@ -523,6 +524,27 @@ func TestPayloadWithEchoBack(t *testing.T) {
 	}
 }
 
+func extractContentLength(body string) int {
```

Comment:
sorry I wasn't clear, I meant to use
https://github.com/istio/fortio/blob/master/fhttp/http_client.go#L38-L40
the 3rd returned parameter which is where does the body start

Reply:
In `TestPayloadWithEchoBack` while I was testing for 3rd parameter, I received a empty body for http 1.0 in fast client. Even I sent body it returns 0. Am I missing flag or is there a bug? Btw I was accually looking for testing fastClient or stdclient

Reply:
The problem that I mention above, I could take a look after this PR

Reply:
do you have something reproducible, I'm not sure what you mean exactly

Reply:
Try this test in http_test in my branch. You will see that `header` will return 0 from `cli.Fetch()`, but it is needed to be 5.
```
func TestFetchError(t *testing.T) {
	var tests = []struct {
		payload           []byte
		http10 bool
	}{
		{[]byte{44, 45, 00, 46, 47}, true},
		{[]byte("groß"), true},
	}
	m, a := DynamicHTTPServer(false)
	m.HandleFunc("/", EchoHandler)
	url := fmt.Sprintf("http://localhost:%d/", a.Port)
	for _, test := range tests {
		opts := NewHTTPOptions(url)
		opts.HTTP10 = test.http10
		opts.Payload = test.payload
		cli := NewClient(opts)
		code, body, header := cli.Fetch()
		if code != 200 {
			t.Errorf("Unexpected error %d", code)
		}
		fmt.Printf("Header size %d\n",header)
		if !bytes.Equal(body[header:], test.payload) {
			t.Errorf("Got %s, expected %s from echo", string(body), string(test.payload))
		}
		cli.Close()
	}
}
```

Reply:
oh http1.0 on purpose doesn't bother parsing headers (to be faster) just don't use that to check POST

Reply:
I released that while I was wrongly using http10 instead of disablefastclient for e2e.